### DISCUSSION TITLE
"Include only" filter

### DISCUSSION THREAD

**Discussion by SantaHey** [deckerst/aves#819] [Ideas]
I would find it cool to have a view to scroll into Camera AND Snapchat medias.
There is an exclude but not an "include only" parameter.
Now I must exclude all others to do this (maybe I don't understand something

I didn't know where to ask, if it's the wrong place you can tell me.

**Comment by deckerst** [comment]
You're right. At the moment the search is based on filters that combine to narrow down the results. They can't be combined to regroup results from different filters. I doubt I will add this feature, because it would complicate the UX/UI for a very niche need.

> I didn't know where to ask, if it's the wrong place you can tell me.

Perfect place to ask ^^
